src/nvim/ops.c

```diff
@@ -2538,18 +2538,18 @@ static void yank_copy_line(yankreg_T *reg, struct block_def *bd, size_t y_idx)
 ///
 /// @param oap Operator arguments.
 /// @param reg The yank register used.
-static void yank_do_autocmd(oparg_T *oap, yankreg_T *reg)
+static void do_autocmd_yank(oparg_T *oap, yankreg_T *reg)
```

Comment:
Shouldn't it be `do_autocmd_textyankpost`, to be consistent with `do_autocmd_dirchanged`